src/Sketches/Datadog.Sketches/Datadog.Sketches.csproj

```diff
@@ -0,0 +1,7 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net461;netstandard2.0;netcoreapp3.1</TargetFrameworks>
```

Comment:
Do we _need_ to choose these specific frameworks? I understand the rationale, just checking whether we want anything else (e.g. `netcoreapp3.0` instead of `netcoreapp3.1`, a `net6.0` target, etc)

Reply:
I think it makes sense to have those because the code will be vendored in the tracer, and those are the targets supported by the tracer.
I don't know if we should add other targets. I really have no argument for or against it 😛